udfs/community/test_cases.js

```diff
@@ -346,6 +346,21 @@ generate_udf_test("typeof", [
         expected_output: `"NUMERIC"`
     },
 ]);
+generate_udf_test("typeof", [
+    {
+        inputs: [`RANGE<DATE> "[null, 2024-04-25)"`],
+        expected_output: `"RANGE<DATE>"`
+    },
+    {
+        inputs: [`RANGE<DATETIME> "[2021-01-01, 2024-04-25)"`],
+        expected_output: `"RANGE<DATETIME>"`
+    },
+    {
+        inputs: [`RANGE(current_timestamp() - interval 5 day, current_timestamp())`],
+        expected_output: `"RANGE<TIMESTAMP>"`
+    },
+
+]);
```

Comment:
```suggestion
generate_udf_test("typeof", [
    {
        inputs: [`RANGE<DATE> "[null, 2024-04-25)"`],
        expected_output: `"RANGE<DATE>"`
    },
]);
generate_udf_test("typeof", [
    {
        inputs: [`RANGE<DATETIME> "[2021-01-01, 2024-04-25)"`],
        expected_output: `"RANGE<DATETIME>"`
    },
]);
generate_udf_test("typeof", [
    {
        inputs: [`RANGE(current_timestamp() - interval 5 day, current_timestamp())`],
        expected_output: `"RANGE<TIMESTAMP>"`
    },
]);
```

Reply:
Done! Thank you @danieldeleo! Let's see if this takes...

Reply:
My tests pass, _i think_, but now getting another failure. In `kruskal_wallis`? What am i doing wrong?